api/src/main/kotlin/com/openlattice/organizations/DatasourcesApi.kt

```diff
@@ -0,0 +1,44 @@
+package com.openlattice.organizations
+
+import retrofit2.http.*
+import java.util.*
+
+
+/**
+ *
+ * @author Matthew Tamayo-Rios &lt;matthew@openlattice.com&gt;
+ */
+interface DatasourcesApi {
+    companion object {
+        // @formatter:off
+        const val SERVICE = "/datastore"
+        const val CONTROLLER = "/organization"
+        const val BASE = SERVICE + CONTROLLER
+        // @formatter:on
+
+        const val ID = "id"
+        const val ID_PATH = "/{$ID}"
```

Comment:
I think it'd be clearer if these are `ORG_ID`.

Reply:
It's ID in organization api and these will be merged into that. Ignore this.